firmware/arduino/chessboard/chessboard.ino

```diff
@@ -96,7 +97,9 @@ void setup()
   pinMode(Y_AXIS_ENDSTOP_SWITCH, INPUT);
   pinMode(CHESS_TIMER_BUTTON, INPUT);
 
+  // Defines different units of board being used
   stepsPerUnitSpace = MILLIMETERS_PER_UNITSPACE * STEPS_PER_MILLIMETER;
+  eighthStepsPerUnitSpace = stepsPerUnitSpace * 8;
```

Comment:
Since we only use eighthStepsPerUnitSpace in a small number of places, we can avoid using a variable for this and just do the assignment directly., like below:
```
  startCol = currentX / (stepsPerUnitSpace * 8);
  startRow = currentY / (stepsPerUnitSpace * 8);
```